q2_feature_table/_summarize/_visualizer.py

```diff
@@ -147,6 +151,14 @@ def summarize(output_dir: str, table: biom.Table,
         fh.write(');')
 
 
+def _compute_qualitative_summary(table):
+    table = table.transpose()
+    sample_count = {}
+    for count_vector, sample_id, metadata in table.iter():
+        sample_count[sample_id] = (count_vector != 0).sum()
```

Comment:
Should this be `sample_count[feature_id]` (I think it's just a variable name change).